src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -343,7 +327,7 @@ public static bool IsGlobalMemberDeclaration(SyntaxKind kind)
             return false;
         }
 
-        public static bool IsNamespaceMemberDeclaration(SyntaxKind kind)
+        public static bool IsTypeDeclaration(SyntaxKind kind)
```

Comment:
note: tihs is a change to teh compiler layer.  tagging @dotnet/roslyn-compiler  @jcouv  for eyes.

Reply:
@dotnet/roslyn-compiler 	any takers?